### DISCUSSION TITLE
Sample Code Feature Request

### DISCUSSION THREAD

**Discussion by malwheer** [alecthomas/participle#367] [Ideas]
Was wondering if you could provide a sample of a STIX parser to find all the file hashes.  (e.g. build a list of all file:hashes.'SHA-256')
Example STIX document: 
https://www.cisa.gov/sites/default/files/2023-09/AA23-201A%20Threat%20Actors%20Exploiting%20Citrix%20CVE%202023-3519%20to%20Implant%20Webshells%20-%20Update.stix_.json

**Comment by alecthomas** [comment]
No, but the existing expression examples are quite similar. It should be straightforward to adapt one of them.

**Reply by malwheer** [reply]
Which one would you recommend I adapt from given the sample link above?
